SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -273,6 +294,37 @@ void ASpatialFunctionalTest::StartTest()
 	StartStep(0);
 }
 
+/*
+ * Adds support for multiple processes testing, see SpatialFunctionalTest.h for more context.
+*/
+void ASpatialFunctionalTest::CallRunTest(const TArray<FString>& Params)
+{
+	if (HasAuthority() == false)
+	{
+		if (LocalFlowController != nullptr)
+		{
+			LocalFlowController->ServerNotifyRunTest(this, Params);
+		}
+		else
+		{
+			FTimerHandle RetriggerTestRunHandle;
+			GetWorldTimerManager().SetTimer(
+				RetriggerTestRunHandle,
+				[Params, WeakThis = TWeakObjectPtr<ASpatialFunctionalTest>(this)] {
+					if (WeakThis.IsValid())
+					{
+						WeakThis->CallRunTest(Params);
```

Comment:
What's the purpose of this 0.1s timer? Has some code smell :D

Reply:
I think this might not be needed, but I'd prefer keeping it as a fail safe-ish. Basically I think at the point of calling StartTest we should always have a flow controller, but I can't 100% guarantee this, maybe @mironec knows more about it.

The timer is just a pattern that we have taken from UFunctionalTestingManager::CheckAndRunTest(). We could instead move it to a tick basis check if you think that would make it less ugly?

Reply:
@mironec do you have context on why this needs to exist buddy? Would like to get some confirmation before approving

Reply:
So the pattern of 0.1s is just to retry this function every 0.1s until a condition succeeds. In this case the flow controller existing. I think it should be fine to keep it this way, since we do want flow controllers here, and I do think it's an async action to spawn them.

Reply:
(theoretically we could remove the checks for flow controllers existing elsewhere, but I think it's minor)

Reply:
👍